gc3libs/backends/batch.py

```diff
@@ -157,6 +165,45 @@ def __init__(self, name,
                 "Unknown transport '%s'" % transport)
         self.accounting_delay = accounting_delay
 
+
+    @property
+    def spooldir(self):
+        """
+        Root folder for all working directories of GC3Pie tasks.
+
+        When this backend executes a task, it first creates a temporary
+        subdirectory of this folder, then launches commands in there.
+
+        If not explicitly set (e.g. at construction time), the "spool
+        directory" will be given a default value according to the logic of
+        :meth:`_discover_spooldir`:
```

Comment:
`self._discover_spooldir` is actually `self._init_spooldir` ?

Reply:
probaly, I simply copied the text from the shellcmd.py script